caos.ansible_roles/roles/install_crowdstrike_falcon/defaults/main.yaml

```diff
@@ -0,0 +1 @@
+win_download_path: 'C:\Users\ansible\Downloads\falcon-sensor.exe'
```

Comment:
Our usecase needs a different download path

Reply:
Why is that?

Reply:
It maybe has to do with the ami itself, but we log in as Admins and there is no other user apparently
<img width="567" alt="image" src="https://github.com/newrelic-experimental/caos-ansible-roles/assets/22451525/dc885f9b-c38f-4e0a-9c60-5de55e75c812">
`vars/main.yml` takes precedence over playbook variables, so I needed to create the CAOS' use case as default 🤔 
It might be a better way to achieve that, though 🙇🏽